spec/features/ineligible_student_loans_claims_spec.rb

```diff
@@ -88,6 +88,8 @@
 
     visit new_claim_path(StudentLoans.routing_name)
 
+    click_on "Start claim"
```

Comment:
I'm still trying to think about what would be the best user journey here. As it stands, when the user gets told they're ineligible, they can visit the start page for that policy again, where they will see the "claim already in progress" message. Clicking "Finish claim in progress" on that screen will take them to the ineligible screen again, so it's not a super-useful action. I wonder whether it would be better to not show the "claim already in progress" screen if the current claim is ineligible?

Reply:
Yeah, good point. I didn't think about that particular case 👍